htmleditor.js

```diff
@@ -41,6 +41,7 @@ class HtmlEditor extends HtmlEditorMixin(SkeletonMixin(LitElement)) {
 	static get properties() {
 		return {
 			disabled: { type: Boolean },
+			insertElement: { type: Boolean },
```

Comment:
```suggestion
			insertElement: { type: Boolean, attribute: 'insert-element' },
```
This ensures we can then set the attribute properly  on the element itself. :)

Reply:
This should be added and pushed!